### ISSUE TITLE
UnicodeHelpersTests.GetDefinedCharacterBitmap_ReturnsSingletonInstance failed in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#1958]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_linux_debug_tst_prtest/657/console
```
Microsoft.Framework.WebEncoders.UnicodeHelpersTests.GetDefinedCharacterBitmap_ReturnsSingletonInstance [FAIL]
09:39:37       Assert.Same() Failure
09:39:37       Expected: UInt32[] [0, 4294967295, 4294967295, 2147483647, 0, ...]
09:39:37       Actual:      UInt32[] [0, 4294967295, 4294967295, 2147483647, 0, ...]
09:39:37       Stack Trace:
09:39:38             at Microsoft.Framework.WebEncoders.UnicodeHelpersTests.GetDefinedCharacterBitmap_ReturnsSingletonInstance()
```

**Comment by KrzysztofCwalina** [comment]
Hmmm, I kept this test (despite removing all other checking sameness) as creating these bitmaps is kind of expensive. But now looking at the code, the code is not doing any kind of synchronization to ensure these are actually singletons. Adding proper synchronization would probably mean simply locking, and I am not sure it's worth it.

**Comment by stephentoub** [comment]
Fixed by #1960
